### ISSUE TITLE
Conflict with rswag UI gem

### ISSUE THREAD

**Issue by kirylpl** [BaseSecrete/rorvswild#17]
Getting http://localhost:3000/api-docs/v1/swagger.yaml casued to an error:

```
Puma caught this error: comparison of String with 200 failed (ArgumentError)
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rorvswild-1.5.7/lib/rorvswild/local/middleware.rb:29:in `>='
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rorvswild-1.5.7/lib/rorvswild/local/middleware.rb:29:in `serve_embed_profiler'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rorvswild-1.5.7/lib/rorvswild/local/middleware.rb:18:in `call'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rorvswild-1.5.7/lib/rorvswild/plugin/middleware.rb:20:in `call'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/sentry-raven-3.0.0/lib/raven/integrations/rack.rb:51:in `call'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/webpacker-5.1.1/lib/webpacker/dev_server_proxy.rb:25:in `perform_request'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/rack-proxy-0.6.5/lib/rack/proxy.rb:57:in `call'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/railties-6.0.3/lib/rails/engine.rb:527:in `call'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/puma-4.3.6/lib/puma/configuration.rb:228:in `call'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/puma-4.3.6/lib/puma/server.rb:713:in `handle_request'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/puma-4.3.6/lib/puma/server.rb:472:in `process_client'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/puma-4.3.6/lib/puma/server.rb:328:in `block in run'
/Users/user/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/puma-4.3.6/lib/puma/thread_pool.rb:134:in `block in spawn_thread'
```

**Comment by kirylpl** [comment]
Without `rorvswild` API docs UI workds like a charm, please take a look.

**Comment by kirylpl** [comment]
Workaround is to put `swagger.yaml` into public folder

**Comment by alexisbernard** [comment]
I have pushed a fix. Can you give it a try and confirm it's working on your side before I publish a new release ?

Gemfile :
```
gem "rorvswild", git: "https://github.com/BaseSecrete/rorvswild.git", ref: "9a27799693988ffb385803a6aabd046ba72422e0"
```

**Comment by kirylpl** [comment]
@alexisbernard It works now, many thanks!

**Comment by alexisbernard** [comment]
Great. I have published version 1.5.8.
